### PR TITLE
Add logger package to workspace one

### PR DESCRIPTION
**Pull request by kylesmartin**
The workspace one logger no longer interferes with the console logger.

### CODE REVIEW

**Comment by redhatrises** [comment]
@kylesmartin looks like there are some unused imports:

```
fig/backends/workspaceone/__init__.py:1:1: F401 'logging.config.dictConfig' imported but unused
fig/backends/workspaceone/__init__.py:2:1: F401 'logging.info' imported but unused
fig/backends/workspaceone/__init__.py:3:1: F401 'ssl.CERT_NONE' imported but unused
fig/backends/workspaceone/__init__.py:3:1: F401 'ssl.PROTOCOL_TLSv1_2' imported but unused
```

**Comment by kylesmartin** [comment]
Should be all set now!

File: fig/backends/workspaceone/serverlog/__init__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from ssl import CERT_NONE, PROTOCOL_TLSv1_2
+from tlssyslog.handlers import TLSSysLogHandler
+from ....config import config
+
+serverlog = getLogger('ws1')
+serverlog.setLevel(INFO)
+
+tlsHandler = TLSSysLogHandler(
+    address=(config.get('workspaceone', 'syslog_host'), int(config.get('workspaceone', 'syslog_port'))),
+    ssl_kwargs={'cert_reqs': CERT_NONE, 'ssl_version': PROTOCOL_TLSv1_2}
```
Comment (isimluk) [inline fig/backends/workspaceone/serverlog/__init__.py:11]: Would be great to make this `CERT_NONE` configurable at some point.

Not topic of this pr, though.

**Review by isimluk — APPROVED** [review]
👍

**Pull request merged**